src/site/content/en/blog/web-animations/index.md

```diff
@@ -0,0 +1,262 @@
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/web-animations/modal-reverse.mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    An example of a modal opening and closing upon button click. <a href="https://codepen.io/kevers-google/details/LYpvdNG">See Demo on Codepen </a>
+  </figcaption>
+</figure>
+
+What you can do is create two play-pending animations (`"openModal"`, and an inline opacity transformation), and then pause one of the animations, delaying it until the other is finished. You can then use promises to wait for each to be finished before playing. Finally, you can check to see if a flag is set, and then reverse each animation.
+
+```js/17-20
+document.querySelector('button').addEventListener('click', () => {                                 
+  // Create two play-pending animations.
+  let animations = [
+    document.querySelector('.modal').animate(openModal, openModalSettings),
+    document.querySelector('.text').animate({ opacity: [0, 1] }, openModalSettings)
+  ];
+
+  // Pause one of the animations and delay it until the other is finished.
+  if (open) {
+    animations.forEach((anim) => {
+      anim.currentTime = anim.effect.getTiming().duration;
+      anim.reverse();
+    });
+    animations = animations.reverse();
+  }
+  
+  // Run animations sequentially.
+  animations[1].pause();  
+  animations[0].finished.then(() => {
+    animations[1].play();
+  });
+  open = !open;
+});
+```
+
+## Dynamic interactions with partial keyframes
+
+<figure class="w-figure">
+<video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/web-animations/retargetting.mp4">
+  </video>  <figcaption class="w-figcaption">
+    Retargeting example, where a mouse click adjusts the animation to a new location. <a href="https://codepen.io/una/details/BaoveQJ">See Demo on Codepen </a>
+  </figcaption>
+</figure>
+
+
+In this example, there is no specified start position. This is an example of using **partial keyframes**. The mouse handler does a few things here: it sets a new end location and triggers a new animation. The new start position is inferred from the current underlying position. 
+
+New transitions can be triggered while existing ones are still running. This means that the current transition is interrupted, and a new one is created.
+
+## Performance improvements with replaceable animations
+
+When creating animations based on events, such as on `"mousemove"`, a new animation is created each time, which can quickly consume memory and degrade performance.  To address this problem, replaceable animations were introduced, enabling automated cleanup, where finished animations are flagged as replaceable and automatically removed if replaced by another finished animation. Consider the following example:
```

Comment:
```suggestion
When creating animations based on events, such as on `'mousemove'`, a new animation is created each time, which can quickly consume memory and degrade performance.  To address this problem, replaceable animations were introduced in Chromium 83, enabling automated cleanup, where finished animations are flagged as replaceable and automatically removed if replaced by another finished animation. Consider the following example:
```
* Using single quotes because that's what the code sample uses
* Adding `in Chromium 84` to make it crystal clear that this is a new feature in v84